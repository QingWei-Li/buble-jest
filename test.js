import transformer from './';

it('es6 to es5', () => {
  expect(transformer.process('() => {}')).toMatchSnapshot();
});

it('ES Modules', () => {
  expect(transformer.process('import a from "./a"; a();')).toMatchSnapshot()
})
