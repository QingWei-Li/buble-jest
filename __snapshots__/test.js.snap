// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ES Modules 1`] = `
"\\"use strict\\";

var _a = _interopRequireDefault(require(\\"./a\\"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

(0, _a.default)();"
`;

exports[`es6 to es5 1`] = `
"\\"use strict\\";

!function() {};"
`;
